client/wildcard/src/components/PageHeader/PageHeader.tsx

```diff
@@ -51,8 +51,13 @@ export const PageHeader: React.FunctionComponent<Props> = ({
         return null
     }
 
+    /// a11y-ignore
+    /// Rule: "landmark-banner-is-top-level" (Banner landmark should not be contained in another landmark)
+    // We should think about stop using `header` here since we couldn't make it be be top-level
+    // with current usages
+
     return (
-        <header className={classNames(styles.container, className)}>
+        <header className={classNames(styles.container, className, 'a11y-ignore')}>
```

Comment:
```suggestion
        <div className={classNames(styles.container, className)}>
```

Let's just keep this as a `div`. Header may be the wrong element here as we can't control where this component is used